Java/crypto/src/main/java/com/breadwallet/crypto/TransferConfirmation.java

```diff
@@ -21,12 +21,16 @@
     private final UnsignedLong transactionIndex;
     private final Date timestamp;
     private final Optional<Amount> fee;
+    private final Boolean success;
+    private final Optional<String> error;
```

Comment:
Pattern in the code base was to not use Optional for fields (but rather @Nullable annotate them). That said, did I use `Optional<Amount> fee` above?

Reply:
This is a top-level, primary interface class.  `Optional` is appropriate here?

Reply:
The guidance I had seen on Optional (and what the code base had been doing) was it was to be only used for return types (and local variables in functions).

If a field or parameter can be `null`, it is annotated with `@Nullable`